inc/Engine/Media/AboveTheFold/WarmUp/Controller.php

```diff
@@ -203,13 +203,14 @@ function ( $link ) use ( $home_url, $reject_uri_pattern ) {
 	 * Send link to SaaS to do the warmup.
 	 *
 	 * @param string $url Url to send.
+	 * @param bool   $is_warm_up set to false by default, should be true if request is warm up.
 	 *
 	 * @return void
 	 */
-	public function send_to_saas( string $url ) {
+	public function send_to_saas( string $url, bool $is_warm_up = false ) {
```

Comment:
is_warm_up might not be a good name: send_to_saas is only use for the warm-up mechanism. I think the point of this bool is more to know if we are in the sync warm-up (for homepage) or the async one (for the other pages)?